articles/network-types-topologies/index.md

```diff
@@ -0,0 +1,91 @@
+Mesh topology increases the network flow and data can be shared faster. It also records a lesser number of network breakages and system breakdowns. But since all the nodes are connected to each other, each node low key acts as a router to another node and so on. This increases power consumption to a larger extent. Mesh topology needs lots of proper planning and requires high maintenance. Due to many links, high network latency may be an issue.
+
+![Mesh Topology](/engineering-education/network-types-topologies/mesh.jpg)
+
+#### Star Topology
+*This topology consists of an extra part, i.e. hub or switch, in the center of the topology which connects all the nodes*. A hub or switch is a networking device that transfers data from one device to another device. It acts as a transmission medium. While both are used depending on their operations, some [differences](https://www.geeksforgeeks.org/difference-between-hub-and-switch/) may affect the way they perform. This is the most commonly used topology.
+In an emergency of network failure, identifying the damaged node becomes easier. This is because a single hub connects all the nodes. Here, the damaged node will be shut down and the rest of the nodes can still be functional. But if a hub is damaged then all the nodes connected to that hub will be shut down for reconstruction.
+
+![Star Topology](/engineering-education/network-types-topologies/star.jpg)
+
+#### Tree Topology
+Tree topology is a special kind of bus topology. Here *central nodes or central hubs are connected to the main bus*. These nodes or hubs are also called as **root nodes**. The other peripheral nodes are connected to those nodes or hubs. These peripheral nodes are called as **descendant nodes**. Like the bus topology, terminators are attached at both ends to define the length of the main bus. This type of topology is used where the data has to be shared in a hierarchical manner.
+If one branch or one descendant node has problems it will not affect the entire network. Adding more branches to the main bus becomes easier. But the main bus connects many sub-branches and if it has any complications, then it affects the whole network. This may also lead to network shut down. Thus, it becomes hard to arrange and group the nodes. Also, it costs very much to install this type of topology.
+
+![Tree Topology](/engineering-education/network-types-topologies/tree.jpg)
+

```

Comment:
A short conclusion would wrap this up nicely - maybe sharing your thoughts on what is the best topology based on what we learned. 
Or perhaps what you believe the next level (of normal) will be for topologies - what innovation or advancement do you for see.

Reply:
Restructured the paragraph(A/D).
Added conclusion part.